go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -382,6 +419,37 @@ func (oc *Controller) createEgressFirewallRules(priority int, match, action, ext
 	return nil
 }
 
+func (oc *Controller) deleteEgressFirewallRule(name string) error {
+	// Find ACLs for a given egressFirewall
+	pACL := func(item *nbdb.ACL) bool {
+		if item.Name != nil {
+			return *item.Name == name
+		}
+		return false
+	}
+	egressFirewallACLs, err := libovsdbops.FindACLsWithPredicate(oc.nbClient, pACL)
+	if err != nil {
+		return fmt.Errorf("unable to list egress firewall ACLs, cannot cleanup old stale data, err: %v", err)
+	}
+
+	if len(egressFirewallACLs) == 0 {
+		return nil
+	} else if len(egressFirewallACLs) > 1 {
```

Comment:
no need for else here since u return above

Reply:
done